db/migrate/20200226115345_delete_always_approve_from_workflows.rb

```diff
@@ -0,0 +1,13 @@
+class DeleteAlwaysApproveFromWorkflows < ActiveRecord::Migration[5.2]
+  def up
+    always_approve = Workflow.find_by(:name => 'Always approve')
+    return unless always_approve
+
+    Request.where(:workflow_id => always_approve.id).update_all(:workflow_id => nil)
```

Comment:
You can let Rails do some of the work for you here but referencing the object and not the `id`.

```ruby
Request.where(:workflow => always_approve).update_all(:workflow => nil)
```

Reply:
Seems `update_all` does not follow relationships

Reply:
Ok, but we can still update the `where` statement.